src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1099,7 +1099,7 @@ void S6Exporter::ExportSpriteVehicle(RCT2SpriteVehicle* dst, const Vehicle* src)
 void S6Exporter::ExportSpritePeep(RCT2SpritePeep* dst, const Peep* src)
 {
     ExportSpriteCommonProperties(dst, static_cast<const SpriteBase*>(src));
-
+    
```

Comment:
Trailing whitespaces on https://github.com/OpenRCT2/OpenRCT2/pull/13735/commits/9d91040273c61bfc239e9246988438a882b1b5b6 (might be solved later down the patches, just want to comment to see)